### ISSUE TITLE
AppStore doesn't work !

### ISSUE THREAD

**Issue by blerand123** [checkra1n/BugTracker#344]
Hi there, 
AppStore doesn't work after jailbreak, ios 13.2.2 , checkra1n 0.9.3.2 
I can't install apps :/

**Comment by Shadowedstalker** [comment]
Try this 

Install iCleanerpro http//ib-soft.net/cydia/
Open iCleanerPro
Click + (lower Left Corner
Click Cydia Substrate Addons
Tap ALL
Tap Apply
Tap yes.

After respring all tweaks are disabled.

Test App Store. If it works it's one of YOUR tweaks causing the issue.

Go back into iCleanerPro and turn your tweaks on one by one, testing the App Store after each one. 
This way you can narrow down what is causing the problem for YOU.

**Comment by blerand123** [comment]
@Shadowedstalker i cant find icleanerpro in cydia can you help me in this one please im suffering to connect to appstore :/

**Comment by Shadowedstalker** [comment]
@blerand123 Did you add http//ib-soft.net/cydia/ to your repos? It's Ivano Bilenchi's repo.

**Comment by Shadowedstalker** [comment]
It appears the repo had changed to https://ib-soft.net/cydia/ .

**Comment by Shadowedstalker** [comment]
http://ib-soft.net/cydia/

**Comment by blerand123** [comment]
@Shadowedstalker, what iam facing here is this: 
can't connect to icloud account, and when i go ex. to install an free app the spinner rotates and stops nothing happens

**Comment by Shadowedstalker** [comment]
open Cydia add https://ib-soft.net/cydia/ to the sources. you may have to respring for anything to show up in the repo.

**Comment by Shadowedstalker** [comment]
Click the site on the mac and it will take you to the repo, but add it to cydia and download from there.

**Comment by blerand123** [comment]
@Shadowedstalker , the iphone had icloud which email was blocked because was my school email and they deleted it after i finished the year, i removed it by following the steps of checkra1n with python script now im unable to install any app on it

**Comment by Shadowedstalker** [comment]
Ok, try this quickly hit volume up, volume down, and then hold the power button. this will Hard Reset the phone (no data loss or anything.) Once the phone restarts you will be in Non-Jailbreak mode ( no Substrate.) try to install an app from the App Store. 

Did it work?

If so rejailbreak the phone and try the App Store again. 

Did it work?

If not, but did before the jailbreak, then it is a tweak you have installed.

**Comment by blerand123** [comment]
@Shadowedstalker, i have used some python commands, and i thinks those commands stops the communication phone-appstore ...

**Comment by Shadowedstalker** [comment]
If that is the case, the only solution you would have is to restore the phone via iTunes and set it up as New, you couldn't use a backup if you have backed it up after you run the commands. I would try the above solution prior to restoring.

**Comment by blerand123** [comment]
@Shadowedstalker if i do that it will be locked again with my deleted icloud acc

**Comment by Shadowedstalker** [comment]
When you say Python Commands are you referring to SSHing into the phone?

It will only have the locked account it you run a backup. not if you set it up as new.

**Comment by blerand123** [comment]
@Shadowedstalker  in root mv /Applications/Setup.app /Applications/Setup.app.crae used this command

**Comment by Shadowedstalker** [comment]
have you tried to reverse the process?

**Comment by Shadowedstalker** [comment]
Was this the Command you used? mv /Applications/Setup.app /Applications/Setup.app.bak I am not familiar with .crae.

**Comment by blerand123** [comment]
it removes from apple center the iphone in order to unlock from icloud passcode

**Comment by Shadowedstalker** [comment]
Can you access the root directory via Terminal again? if so see if you have /Applications/Setup.app and /Applications/Setup.app.crae? If so you may be able to rm the /Applications/Setup.app.crae. 

However a System restore would most likely fix it too.

**Comment by Shadowedstalker** [comment]
It's never a good idea to try and bypass apple system files without knowing exactly what will happen. it's also better to rename files .bak, .old, so that if a problem happens you can always revert to the original file.

**Comment by Shadowedstalker** [comment]
An example of this would be, I hate the power connected sound. Instead of searching Cydia, I went to /System/Library/Audio/UISounds and renamed connect_power.caf to connect_power.caf.old, easily revertible.

**Comment by blerand123** [comment]
./tcprelay.py -t 44:2222
ssh root@localhost -p 2222
mount -o rw,union,update /
echo "" >> /.mount_rw
mv /Applications/Setup.app /Applications/Setup.app.crae
uicache --all  
killall backboardd

these are the commands that i did on terminal

**Comment by blerand123** [comment]
@Shadowedstalker

**Comment by Shadowedstalker** [comment]
Can you post your source for this?

I can see mv /Applications/Setup.app /Applications/Setup.app.bak you I can't find anything with you command.

**Comment by Shadowedstalker** [comment]
I found this on reddit 

Open Terminal Window and Type the following: iproxy 2222 44

Open another Terminal window and type the following: ssh root@localhost -p 2222

Type commands in order when connected to device:

mount -o rw,union,update /

mv /Applications/Setup.app /Applications/Setup.app.bak

uicache --all

killall backboardd

And thats it! Enjoy!

**Comment by blerand123** [comment]
so i dont need to use ./tcprelay.py -t 44:2222 this right ?

**Comment by blerand123** [comment]
@Shadowedstalker after i rejailbreak to use the same commands you just sent to me right ?

**Comment by Shadowedstalker** [comment]
My concern is 

mv /Applications/Setup.app /Applications/Setup.app**.crae** <- yours

mv /Applications/Setup.app /Applications/Setup.app**.bak** <-- the post.

**Comment by blerand123** [comment]
hmm i will try it on my another 5se to see if will work, so i will not break anything on my iphone x

**Comment by Shadowedstalker** [comment]
Are you facing the same problem on both devices?

**Comment by blerand123** [comment]
yes

**Comment by Shadowedstalker** [comment]
Any luck?

**Comment by CySxL** [comment]
Hey! I've got the same issue!
Can't download new app in appstore but can upgrade.

Even when I'm in non-jailbroken mode, still can't. 

The app download/get button just loading in a circle in a second and return to the download/get view after tapping on it.

I've try iCleanerPro, no luck.
And reboot to non-jailbroken mode, still can't.

And I'm not using any tweak related to Appstore ,after finding out this issue, i try Appstore++ to use downgrade button and still showing no version to download.

I'm using iPhoneX, on iOS13.3, latest Checkra1n(Bootra1n) with verbose boot option turning on.

**Comment by dachengxi** [comment]
I've the same issue!
iphone 6 ios 12.4 
after i use checkra1n  0.9.8.1 beta, appstore can't use, apps can't open, itunes can't login. I reboot my iphone, it's still not working.

i tried @Shadowedstalker way , still not working.

there is anyone solved this?

**Comment by anxietyd** [comment]
@CySxL do u fix it? I've got the same issue :(

**Comment by anxietyd** [comment]
Well. Just download “icleaner pro” and enable all daemons (“+” -> “iOS functions”)
![AE0CF2D2-E173-4995-A775-D716875A3149](https://user-images.githubusercontent.com/72986807/96283984-07a7c280-0fee-11eb-8941-9dc159cab1e0.png)
.

**Comment by XXxxxD3ATHxxxXX** [comment]
Just reset all settings on ur iPhone and it will work trust me on this It happened to me as well

**Comment by anxietyd** [comment]
@XXxxxD3ATHxxxXX no, I needed to save app data from a backup that got corrupted by jailbreak. had to look for a solution.

**Comment by CySxL** [comment]
> @CySxL do u fix it? I've got the same issue :(
@anxietyd I solved it

by logging out Apple ID and logging in again
and it will ask u something like remember ur installation (or password I guess, too long ago for me haha), reject all those alert popping up and you'll be fine !

**Comment by lexieye** [comment]
> Hi there,
> AppStore doesn't work after jailbreak, ios 13.2.2 , checkra1n 0.9.3.2
> I can't install apps :/

Jailbroken Checkra1n Latest Release
checkra1n 0.12.4 beta via Linux command on pc with Raspberry Pi OS for PC and Mac.
 Iphone MN0W2B iOS 14.0.1

I have to reboot and verify payment method to use AppStore (after every jailbreak)

**Comment by Veshurik** [comment]
I have same problem, but on iOS 10.3.4 (helix). 
I don't know what causes the problem, and iCleanerPro doesn't have Cydia Substrate add-ons button
